HdmiCecSink/HdmiCecSink.cpp

```diff
@@ -888,7 +898,39 @@ namespace WPEFramework
             params["audioMode"] = msg.status.toString().c_str();
             sendNotify(eventString[HDMICECSINK_EVENT_SYSTEM_AUDIO_MODE], params);
          }
+         void HdmiCecSink::Process_ReportAudioStatus_msg(const ReportAudioStatus msg)
+         {
+            JsonObject params;
+            if(!HdmiCecSink::_instance)
+               return;
+			LOGINFO("Command: ReportAudioStatus  %s audio Mute status %d  means %s  and current Volume level is %d \n",GetOpName(msg.opCode()),msg.status.getAudioMuteStatus(),msg.status.toString().c_str(),msg.status.getAudioVolume());
+            params["muteStatus"]  = msg.status.getAudioMuteStatus();
+            params["volumeLevel"] = msg.status.getAudioVolume();
+            sendNotify(eventString[HDMICECSINK_EVENT_REPORT_AUDIO_STATUS], params);
 
+         }
+		 void HdmiCecSink::sendKeyPressEvent(const int logicalAddress, int keyCode)
+		 {
+		    LOGINFO(" sendKeyPressEvent logicalAddress 0x%x keycode 0x%x\n",logicalAddress,keyCode);
+                    switch(keyCode)
```

Comment:
No need to define VOLUME_UP VOLUME_DOWN etc as we are anyway asking UICommand::UI_COMMAND_VOLUME_UP from UI so below three switch cased can be clubbed in one